src/editors/containers/TextEditor/components/ImageSettingsModal/__snapshots__/index.test.jsx.snap

```diff
@@ -107,6 +121,16 @@ exports[`ImageSettingsModal render snapshot 1`] = `
                   },
                 },
               },
+              Object {
```

Comment:
Nothing should have been added to the dimensioncontrols object right?

Reply:
Even if something changed about onImgLoad, DimensionControls doesn't read that value anyways.

Reply:
Yeah v confused-- we might have to did into what the test is doing here.